app/scripts/proactive/config.js

```diff
@@ -62,6 +62,52 @@ define(function () {
             'Script/flow' : [" ", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "powershell", "R"],
             'Script/clean' : [" ", "bash", "cmd", "kubernetes", "docker-compose", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R"]
         },
+        extensions_to_languages: {
+            "sc": "scalaw",
+            "scala": "scalaw",
+            "groovy": "groovy",
+            "js": "javascript",
+            "cpy" : "python",
+            "py" : "jython",
+            "rb": "ruby",
+            "ps": "powershell",
+            "r": "R",
+            "sh": "bash",
+            "bat": "cmd",
+            "yaml": "kubernetes",
+            "yml": "docker-compose",
+            "pl": "perl"
+        },
+        languages_to_extensions: {
+            "scalaw" : "scala",
+            "groovy" : "groovy",
+            "javascript" : "js",
+            "python" : "py",
+            "cpython" : "py",
+            "ruby" : "rb",
+            "r" : "r",
+            "bash" : "sh",
+            "cmd" : "bat",
+            "kubernetes" : "yaml",
+            "docker-compose" : "yml",
+            "perl" : "pl",
+            "powershell" : "ps"
+        },
+        languages_content_type: {
```

Comment:
you don't use this map anywhere in the code.

Reply:
Thanks, I forgot to set the content type depending on the language. This is done now